ui/src/Annotation.tsx

```diff
@@ -100,7 +109,8 @@ export class Annotation extends React.PureComponent<
                     "scholar-reader-annotation",
                     this.props.className,
                     {
-                      selected: this.isSelected()
+                      selected: this.isSelected(),
+                      "annotation-highlight": this.shouldHighlight(),
```

Comment:
I feel like `annotation-highlight` is a bit vague here, I wonder if we can give it a more semantic class name. Say, something like `matching-symbol-annotation`?

Reply:
I like matching-symbol-annotation!